sceptre/scipool/README.md

```diff
@@ -31,15 +31,44 @@ Requirements:
 * Run `pre-commit install` to install the git hook.
 
 ## Testing
+
+### Pre-Commit
 As a pre-deployment step we syntatically validate our sceptre and
 cloudformation yaml files with [pre-commit](https://pre-commit.com).
 
 Please install pre-commit, once installed the file validations will
-automatically run on every commit.  Alternatively you can manually
+automatically run on every commit. Alternatively you can manually
 execute the validations by running `pre-commit run --all-files`.
 Please install pre-commit, once installed the file validations will
 automatically run on every commit.
 
+### Functional Testing
+The process to test the functionality of an AMI and it's integration with our
+Service Catalog products is to first create a test AMI, upload a modified
+product template to S3, and create a new Service Catalog product in the
+scipool dev account to verify manually from https://sc-dev.sageit.org/
+
+The deploy pipelines for both our packer repos and our service catalog library
+will create artifacts for branches that begin with `test/` in the sandbox
+account, allowing anyone with write access to the packer repos to create test
+AMIs, and anyone with write access to service-catalog-library to upload test
+templates for service catalog to S3.
+
+1. Commit changes to the packer repo to update or modify the AMI on a branch
+   that starts with `test/`, and push directly to the origin repo.
+1. Manually create an EC2 instance in the `itsandbox` account from the test AMI
+   for any initial system validation, then terminate it.
+1. Commit changes to `service-catalog-library` on a branch that starts with
+   `test/` to update the desired template, and push directly to the origin repo.
+1. Create a pull request for `organizations-infra` to add a new Service Catalog
+   product to `scipool-dev` with 'test' in the name for the test template.
```

Comment:
Can you mention where the "product" templates are located in the repo'?  How does the product reference the template in the previous step?

Reply:
The wording is a little tricky because there's a template in service-catalog-library, and another in organizations-infra. I've added a link to an example PR that hopefully shows the relationship between the two templates.